packages/eventstream-handler-node/src/EventStreamPayloadHandler.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { Readable, PassThrough } from "stream";
+const mockSingingStream = jest.fn().mockImplementation(() => new PassThrough());
+jest.mock("./EventSigningStream", () => ({
+  EventSigningStream: mockSingingStream
+}));
+import { EventStreamPayloadHandler } from "./EventStreamPayloadHandler";
+import {
+  EventSigner,
+  Decoder,
+  Encoder,
+  FinalizeHandler,
+  HttpRequest,
+  FinalizeHandlerArguments
+} from "@aws-sdk/types";
+
+describe("EventStreamPayloadHandler", () => {
```

Comment:
Add newline between `it()` blocks for better readability

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/1082/commits/8e9bbc011160d4bf0fd0e799cf4873a875d95c12